docs/using.rst

```diff
@@ -140,7 +140,13 @@ Sometimes the tests that I am developing can be destructive to the system so I
 don't want to run them on my development box. Or the test is specific to an
 architecture so I can't use a VM for it on my machine. These are cases where
 it's really handy to use a combination of Beaker for provisioning and
-Standalone for executing the tests.
+Standalone for executing the tests. By default, Beaker provides a test harness
+for all imported distributions. You can replace test harness with your build
+by adding a new repository.You can create your build on your own or you can
```

Comment:
Missing space between "repository.You"

Reply:
Wow, you have good :eyes: . Didn't noticed. Fixed.